types/mirrorx/mirrorx-tests.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import mirror, { actions } from 'mirrorx';
```

Comment:
```ts
import mirror = require('mirrorx');
```

Then later below, instead of `actions` you would use `mirror.actions`

Reply:
@minestarks sorry to disturb you. I have fixed all the errors in the lint. Please check.

Reply:
@minestarks Could you check it please?